src/database/schema/ChatSchema.ts

```diff
@@ -251,11 +279,11 @@ class ChatSchema implements BaseDbSchema {
     return new ChatSchema({
       channelId,
       message,
-      status: 'pending',
+      status,
       createdAt: new Date().toISOString(),
       updatedAt: new Date().toISOString(),
       id: newRandomId,
-      type: 'regular',
+      type,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The properties `status`, `createdAt`, `updatedAt`, `id`, and `type` are now set using the parameters passed to the function instead of hardcoded values. Make sure that all calls to this function pass the correct values for these parameters.
```diff
-      status: 'pending',
-       createdAt: new Date().toISOString(),
-       updatedAt: new Date().toISOString(),
-       id: newRandomId,
-      type: 'regular',
+      status,
+       createdAt: new Date().toISOString(),
+       updatedAt: new Date().toISOString(),
+       id: newRandomId,
+      type,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->